dev-server/oidc/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "mock-oidc",
+  "version": "0.1.0",
+  "main": "server.js",
+  "engines": {
+    "node": ">=10.15.0 <12"
```

This feels a bit outdated, the current LTS relase is 12. This forces the use of an older version.
```suggestion
    "node": ">=12"
```